templates/data/kyma-telemetry-data.yaml

```
@@ -0,0 +1,33 @@
+templates:
+  - fromTo:
+      - from: templates/generic.tmpl
+        to: ../prow/jobs/kyma/components/telemetry-operator/telemetry-test.yaml
+    render:
+      - localSets:
+          jobConfig_default:
+            path_alias: github.com/kyma-project/kyma
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/kyma-telemetry-test.sh"
+        jobConfigs:
+          - repoName: "kyma-project/kyma"
+            jobs:
+              - jobConfig:
+                  name: "telemetry-operator-test"
+                  run_if_changed: '^components/telemetry-operator/|^resources/telemetry/'
+                inheritedConfigs:
+                  global:
+                    - "jobConfig_default"
+                    - "jobConfig_presubmit"
+                    - "jobConfig_postsubmit"
+                    - "image_buildpack-golang-toolbox"
+                  local:
+                    - "jobConfig_default"
```

If you want to define both pre- and postsubmit jobs in one see https://github.com/kyma-project/test-infra/blob/main/templates/data/kyma-validation-data.yaml#L30 as an example
```suggestion
              - jobConfig:
                  run_if_changed: '^components/telemetry-operator/|^resources/telemetry/'
                jobConfigPre:
                  name: "pre-telemetry-operator-test"
                jobConfigPost:
                  name: "post-telemetry-operator-test"
                inheritedConfigs:
                  global:
                    - "jobConfig_default"
                    - "image_buildpack-golang-toolbox"
                  preConfigs:
                    global:
                      - "jobConfig_presubmit"
                  postConfigs:
                    global:
                      - "jobConfig_postsubmit"
                  local:
                    - "jobConfig_default"
```

---

Nice, thanks! :)